content/en/docs/setup/platform-setup/openshift/index.md

```diff
@@ -9,6 +9,10 @@ aliases:
 keywords: [platform-setup,openshift]
 ---
 
+{{< warning >}}
+OpenShift uses `nftables`, which is incompatible with the Istio proxy-init container. Please use [CNI](/docs/setup/additional-setup/cni/) instead.
```

Comment:
The nftables support has not been merged. Istio CNI as shipping in Istio 1.3.0 will use iptables.

Reply:
Istio CNI is using the `iptables` command installed on the worker node, so it may have been using the `nftables`'s `iptables` compatibility layer that may have been installed on your worker nodes. Is this guaranteed to be available on every OpenShift node?

Reply:
This is 4.1 and above. Clarifying the statement. On those systems, the `iptables` in proxy-init fails (see: https://github.com/istio/istio/issues/13986). In the case of CNI, it uses the host's `iptables` command which is the `nftables` compatibility layer you referenced.

Reply:
Thanks